src/DragDrop/DragDrop.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { Card, ProgressBar, Button } from "react-bootstrap";
+import { useDropzone } from "react-dropzone";
+import { storage } from "../Firebase/Firebase";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./styles.css";
+
+export default function DragDrop() {
+  const [files, setFiles] = useState([]);
+  const [progress, setProgress] = useState(0);
+
+  // Initialise dropzone
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    accept: "image/*",
+    onDrop: (acceptedFiles) => {
```

Comment:
Concern for the future: right now it is possible to upload .gif files.
Maybe limit the file types to png, jpg and jpeg only.

But of course this would be a nice to have.